java/configuration/src/main/java/sleeper/configuration/properties/instance/DefaultProperty.java

```diff
@@ -131,9 +131,9 @@ public interface DefaultProperty {
             .propertyGroup(InstancePropertyGroup.DEFAULT).build();
     UserDefinedInstanceProperty DEFAULT_INGEST_BATCHER_INGEST_MODE = Index.propertyBuilder("sleeper.default.ingest.batcher.ingest.mode")
```

Comment:
Now `BatchIngestMode` no longer exists, maybe this can be called `DEFAULT_INGEST_BATCHER_INGEST_QUEUE` or `DEFAULT_INGEST_BATCHER_DESTINATION_QUEUE` or something similar?

Reply:
Done

Reply:
The property key also needs updating here, the templates would need regenerating too

Reply:
That's a breaking change, but I've done that now.